fern/docs/private-deployment/octostack.mdx

```diff
@@ -23,6 +23,31 @@ OctoStack includes OctoAI’s performance optimizations delivered through Apache
 ### Easy to use APIs & customization
 Application developers can use OctoStack’s industry-standard API’s, including Python and Typescript SDK’s. Our ergonomic API’s allow rapid development and integration. You can easily load and run open source models, and fine-tuned checkpoints.
 
+# Configuration & Deployment
+
+### GPU Configuration
+
+OctoStack can run a multi-GPU configuration to improve throughout and latency, which is easily enabled with a single configuration setting. We recommend using multiple GPU’s per replica when running models with higher precision and a greater number of parameters. The OctoAI team can help you identify the multi-GPU configuration that best matches your throughput & latency goals. 
+
+### Prerequisites
+
+OctoStack is comptable with all cloud environments - these are key system requirements:
+
+- Kubernetes, or Docker & Docker Compose
+- Redis in cluster mode
+- NVIDIA drivers
+- GPU’s
+    - OctoStack supports a range of hardware including A10G, A100, and H100 GPU’s
+    - A100’s such as AWS p4d.24xlarge instances can support a broad set of models
+
+### Deployment
+
+OctoStack supports Kubernetes deployment via manifest files, and a Docker Compose deployment compatible with any container orchestration service. There’s a few simple steps to deploy:
+
+1. OctoAI allow lists your AWS account, so you can access OctoStack containers in OctoAI’s AWS ECR.
```

Comment:
This is phrased confusingly to me -- do you mind adding some clarity?

Reply:
sure, I will spin up a follow on change for this